### ISSUE TITLE
[Feature Request] An option to sort by folders

### ISSUE THREAD

**Issue by alex285** [GabMus/HydraPaper#13]
since you can add several folders, it would be nice if there was an option to be able to sort by locations, similar to Favorites

for example

Favorites ----------------------------------

Anime     ------------------------------ (thats folder name)

Scenery  ------------------------------------ (thats folder name)

Folders shouldn't scroll on a different container though like Favorites ..which btw im not either sure that Favorites should scroll on a different box. a better way is to have a hide/show button next that collapses them -in case someone has 150 favorites :)

**Comment by GabMus** [comment]
Sorting by folder could be useful actually, but I'd want to keep it as simple as possible, I don't want to make a file manager.

I could also implement a way to hide favorites, maybe using a stacked view and putting them in another page completely?

Let me know what you think.

**Comment by GabMus** [comment]
Migrated the interface to a stacked view, awaiting feedback

**Comment by alex285** [comment]
@GabMus  
> I don't want to make a file manager.

i agree, thats why i said only as option, and only if isn't very complex. to me it makes sense because i categorize the wallpapers in folders - pro wallpapering :)

>  awaiting feedback

i will try a bit later when login to Arch, because im on Ubuntu now :/

**Comment by GabMus** [comment]
Didn't I send a flatpak? Sorry here you go

[hydrapaper-5d4e439.zip](https://github.com/GabMus/HydraPaper/files/1688700/hydrapaper-5d4e439.zip)

**Comment by alex285** [comment]
@GabMus sorry for the late response! it is better now with the stacked view, but one problem is that when you favorite a wallpaper it disappears from the "All" view. i believe it should remain there, and have a "Heart" overlay icon, so you can easily look  all your wallpaper in one place

**Comment by GabMus** [comment]
Added the option to keep favs in mainview, please test and report back.

~~[hydrapaper-d6fae46.zip](https://github.com/GabMus/HydraPaper/files/1698232/hydrapaper-d6fae46.zip)~~

**Comment by GabMus** [comment]
New snapshot with a little patch since `d6fae46`.

[hydrapaper-f7d54c8.zip](https://github.com/GabMus/HydraPaper/files/1698287/hydrapaper-f7d54c8.zip)

**Comment by alex285** [comment]
@GabMus nice one! can you add a "heart" icon on images that are already favorites?

**Comment by GabMus** [comment]
>  can you add a "heart" icon on images that are already favorites

yes, but will require a little bit of time

**Comment by GabMus** [comment]
Added heart icon

[hydrapaper-9e0feb8.zip](https://github.com/GabMus/HydraPaper/files/1699185/hydrapaper-9e0feb8.zip)

**Comment by alex285** [comment]
@GabMus that was fast! so much thank you!

**Comment by GabMus** [comment]
I was thinking about this, and when I tried to implement it, I figured out it was going to be messy af. doesn't the hide/show folder feature suffice?
